Vizzuality/gfw
# Feature/header component

## Overview

This is a preliminary feature to prepare for updating the header component with the new design changes. This swaps out the local header component for the one supplied by the detached component library that all other apps are using.

Some BONUS fixes:
- home page carousels now showing correct number of slides in mobile
- Update metadata title for search page
- Improved search interactions by using the search component from `gfw-components`

## Testing

- check that the header shows
- click links and make sure they route like an SPA
- make a search in the sub menu panel and confirm it works in the search page
- change the language and make sure you see string changes and not errors
- open the contact us modal from the submenu panel
- open the menu in mobile and make sure the correct links are there
